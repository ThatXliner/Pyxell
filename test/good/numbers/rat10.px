
print 9223372036854775808
print -0x1_0000_0000_0000_0000 / 3
print 10^^18 / 1_0000000000_0000000000
