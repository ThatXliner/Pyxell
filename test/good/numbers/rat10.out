9223372036854775808/1
-18446744073709551616/3
1/100
